### PR TITLE
Chapter 7 update

### PR DESCRIPTION
**Pull request by pauldpickell**
Draft Chapter 7 with images and data

### CODE REVIEW

File: 07-topology-and-geocoding.Rmd [hunk 1]
```diff
@@ -20,36 +20,76 @@ Frequently, we need spatial data to behave and relate and in specific and predic
 
 ### Key Terms {-}
 
-Vertex, Node, Pseudonode, Dangle, Planar Topology, Non-Planar Topology, Geocoding, Adjacency, Overlap, Connect, Inside, Reverse Geocoding
+Vertex, Node, Pseudonode, Dangle, Planar Topology, Non-Planar Topology, Geocoding, Adjacency, Overlap, Connect, Inside, Reverse Geocoding, small change
```
Comment (ect123) [inline 07-topology-and-geocoding.Rmd:23]: Not sure if last keyword ("small change") is meant to be included of if is a commit message that was added to the wrong place.

File: 09-raster-analysis-terrain-modelling.Rmd [hunk 2]
```diff
@@ -275,7 +275,7 @@ One of the motivations for deriving information from a DEM is to classify terres
...
+Curvature of terrain is described as convex, flat, or concave, which impacts the flow acceleration or deceleration of runoff over the terrain. There are two components of terrain curvature representing the x- and y-axes of geographic space. **Profile curvature** describes the downslope curvature while **plan or planform curvature** describes the curvature that is perpendicular to the downslope curvature. In this way, the vertical expression of terrain can be described by two, perpendicular curves. Figure \@ref(fig:9-stacked-profile-planform) illustrates the nine different combinations of profile and planform curvature. change
```
Comment (ect123) [inline 09-raster-analysis-terrain-modelling.Rmd:278]: The last word in this paragraph is oddly placed, and I'm not sure if it belongs.

File: 07-topology-and-geocoding.Rmd (hunk 1 above)
Comment (pauldpickell) [reply 07-topology-and-geocoding.Rmd:23]: Odd, not sure how that ended up in there. Ditto for comment below.

**Review by ect123 — APPROVED** [review]
I've left two comments for very small edits which can be addressed in later reviews.

**Pull request merged**